azure-pipelines.yml

```diff
@@ -1,5 +1,9 @@
 # Azure Pipelines CI build configuration
 # Documentation at https://aka.ms/yaml
+#
+# Copyright lowRISC contributors.
```

Comment:
Small nit before hopefully returning to a more in depth review: for this and similar files, I think we'd be more consistent to have the license header as the very first thing in the file - I think our standard is to have it as early in the file a possible (first line, or after the shebang normally)

Reply:
The tool accepts "within the first contiguous-ish comment" (you'll see I had to reformat a block comment to support the tool), but I'm happy for the additions I make in this PR to be made as near to the top as possible (given shebangs and the format of `.core` files).